buildSrc/private/src/main/kotlin/androidx/build/AndroidXComposeMultiplatformExtensionImpl.kt

```diff
@@ -262,4 +266,30 @@ open class AndroidXComposeMultiplatformExtensionImpl @Inject constructor(
             iosSimulatorArm64("uikitSimArm64") { configureFreeCompilerArgs() }
         }
     }
+
+    override fun configureTestsRunInIosSimulatorEnvironment(device: String?): Unit =
```

Comment:
We need to be ready to combine `AndroidXCompose` copy back into `AndroidXExtension` (used in AOSP and in fork for lifecycle modules).

Please do that changes in `AndroidXPlugin` instead of `AndroidXComposePlugin` (or at least in both places).

Reply:
After discussion with @eymar , we decided to place it here. Let's discuss to have clear understanding.